Assets/Tests/InputSystem/Plugins/OnScreenTests.cs

```diff
@@ -263,6 +263,27 @@ public IEnumerator Devices_CanHaveOnScreenJoystickControls()
         Assert.That(Gamepad.all[0].buttonSouth.isPressed, Is.False);
     }
 
+    // https://fogbugz.unity3d.com/f/cases/1305016/
+    [Test]
+    [Category("Devices")]
+    public void Devices_CanUseKeyboardCurrentAfterDisablingOnScreenButton()
```

Comment:
Recommend not making this specific to on-screen controls but rather just have a generic test `Devices_RemovingKeyboardMakesNextKeyboardCurrent` along with the other device tests.

Reply:
I'll add a keyboard test separately. But would like to keep bug specific test, just so we keep sure that something as disabling `OnScreenButton` doesn't break keyboard API's.

Reply:
Added `Devices_RemovingKeyboardMakesNextKeyboardCurrent`